Library/Homebrew/help.rb

```diff
@@ -72,6 +72,9 @@ def help(cmd = nil, flags = {})
       # Resume execution in `brew.rb` for unknown commands.
       return if path.nil?
 
+      # Resume execution in `brew.rb` for external commands using "cli/parser".
+      return if !flags[:internal_cmd] && path.read.match?("require .cli/parser.")
```

Comment:
Would love to figure out something else beyond reading the file here. Maybe require the file and look for the relevant method name (which could/should be part of `cli/parser` itself if it isn't already)?

Reply:
Something like that:

```ruby
require path
methods.include?(:usage_banner)
```

?

Reply:
Yeh, `:announce_args` or similar. Would be good to share logic with https://github.com/Homebrew/brew/blob/e47a6ac4cb8810ab687096f224e1fc4c0f0f980e/Library/Homebrew/dev-cmd/man.rb#L156-L166

Reply:
Okay, gonna need more help with that :smile: